lib/ansible/modules/identity/keycloak/keycloak_client.py

```diff
@@ -729,6 +731,9 @@ def main():
                 new_param_value = sorted(new_param_value)
             except TypeError:
                 pass
+            finally:
+                if PY2:
+                    sys.exc_clear()  # Avoid false positive traceback in fail_json() on Python 2
```

Comment:
This looks like it would be good as long as it was in the TypeError instead of in the finally.